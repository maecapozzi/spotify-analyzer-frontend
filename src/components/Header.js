import React from 'react'
import PropTypes from 'prop-types'
import injectSheet from 'react-jss'

const styles = {
  h1: {
    color: '#BFDBF7',
    fontFamily: 'Inconsolata',
    fontWeight: '300',
    letterSpacing: '3px',
    textAlign: 'center',
    marginTop: '5em'
  }
}

const Header = ({ classes, string }) => <h1 className={classes.h1}>{string}</h1>

Header.propTypes = {
  string: PropTypes.string
}

export default injectSheet(styles)(Header)
