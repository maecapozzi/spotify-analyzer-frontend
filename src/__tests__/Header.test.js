import React from 'react'
import { shallow } from 'enzyme'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-15'

import Header from '../components/Header'

configure({ adapter: new Adapter() })

test('it renders correctly', () => {
  const component = shallow(<Header />)
  expect(component).toMatchSnapshot()
})
