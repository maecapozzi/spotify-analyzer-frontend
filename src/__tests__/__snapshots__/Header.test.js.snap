// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Jss(Header) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "h1": "Header-h1-0-1",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "h1": "Header-h1-0-1",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;

exports[`renders correctly 1`] = `
<h1
  className="Header-h1-0-1"
/>
`;
